Improve TiDB Cloud MFA function (#20567)
pingcap/docs: tidb-cloud/tidb-cloud-password-authentication.md

3. Click the **Change Password** tab, click **Change Password**, and then check your email for TiDB Cloud to reset the password.

> ## Manage Multi-Factor Authentication (optional)

Reviewer: ```suggestion
## Manage multi-factor authentication (optional)
```

> **Note:**
>
> - This section applies only when you [sign up](https://tidbcloud.com/free-trial) for TiDB Cloud with emails and passwords. If you sign up for TiDB Cloud with Google, GitHub, or Microsoft SSO, you can enable MFA on your chosen identity management platform.

> > - If you have enabled TiDB Cloud MFA in the SSO login scenario, to ensure the security of your account, please migrate your MFA management mechanism to your SSO identity management platform before September 30, 2025.

Reviewer: ```suggestion
> - If you have enabled TiDB Cloud MFA in an SSO login scenario, migrate your MFA management to your SSO identity management platform before **September 30, 2025**, to ensure account security.
```

> Multi-Factor authentication (MFA) adds additional security by requiring an Authenticator app to generate a one-time verification code for login. During login, TiDB Cloud needs to verify your password and MFA verification code. Regarding the authenticator app, You can use any Authenticator app from the iOS or Android App Store to generate this password, such as Google Authenticator and Authy.

Reviewer: ```suggestion
Multi-factor authentication (MFA) adds additional security by requiring an Authenticator app to generate a one-time verification code for login. When you log in, TiDB Cloud verifies both your password and the MFA verification code. You can use any Authenticator app from the iOS or Android App Store to generate this password, such as Google Authenticator and Authy.
```

### Enable MFA

1. Click <MDSvgIcon name="icon-top-account-settings" /> in the lower-left corner of the TiDB Cloud console.

2. Click **Account Settings**.

3. Click the **Multi-Factor Authentication** tab.

4. Click **Enable**.

> 5. Enter your account password to ensure security.

Reviewer: ```suggestion
5. Enter your account password to confirm your identity.
```

> 6. Go to the **Set Up Authenticator App** section, where you will need to use your authenticator app to scan the QR code to associate the MFA device.

Reviewer: ```suggestion
6. In the **Set Up Authenticator App** section, use your authenticator app to scan the QR code and associate your MFA device.
```

> 7. Enter the authentication code generated by your app to complete the MFA device association operation.

Reviewer: ```suggestion
7. Enter the authentication code generated by your app to complete the MFA device association.
```

> 8. You will receive a one-time recovery code. It can help you perform an MFA authentication in case your MFA device is unavailable.

Reviewer: ```suggestion
8. Save the one-time recovery code, which helps you authenticate if your MFA device is unavailable.
```

> **Note:**
>

> > - You need to keep the recovery code properly to ensure the security of your account.
> > - If you use the recovery code for MFA verification when logging in, the system will automatically generate a new recovery code for you after the verification is successful.

Reviewer: ```suggestion
> - You need to store the recovery code securely to maintain account security.
> - If you use the recovery code for MFA verification during login, the system automatically generates a new recovery code after successful verification.
```

### Disable MFA

1. Click <MDSvgIcon name="icon-top-account-settings" /> in the lower-left corner of the TiDB Cloud console.

2. Click **Account Settings**.

3. Click the **Multi-Factor Authentication** tab.

4. Click **Disable**.

> 5. Go to the **Disable Multi-Factor Authentication** section, where you will need to fill in your account password and MFA authentication code to ensure the security of the operation.

Reviewer: ```suggestion
5. In the **Disable Multi-Factor Authentication** section, enter your account password and MFA authentication code to confirm the operation.
```

> **Note:**
>

> > Be cautious when disabling MFA operations, as this will reduce the security of your account.

Reviewer: ```suggestion
> Disabling MFA reduces the security of your account. Proceed with caution.
```
